packages/discovery-react-components/README.md

```diff
@@ -53,3 +63,10 @@ const App = () => {
   );
 };
 ```
+
+## Setting up PdfViewer
+
+To use the PdfViewer component (or the components that use it, e.g. PdfViewerWithHighlight, DocumentPreview) you will need to provide pdf.js with a PDF worker. This can be done in one of two ways:
+
+- Use the `pdfWorkerUrl` prop to send the URl of a pdf worker JS file (either self-hosted or externally hosted) to any of `PdfViewer`, `PdfViewerWithHighlight`, `DocumentPreview`, or `DocumentPreview.PreviewDocument`.
+- Set `GlobalWorkerOptions.workerSrc` globally (see `src/setupTests.ts` for an example).
```

Comment:
```suggestion
## Rendering PDFs

If you want to the Discovery Components (`DocumentPreview` or `PdfViewer`) to render PDF documents, you will need to set up the pdf.js worker script. This can be done in one of two ways:

1. Set the `pdfWorkerUrl` prop to the URL of the pdf.js worker script (i.e. `pdf.worker.min.js`) to any of `DocumentPreview`, `PdfViewer`, `PdfViewerWithHighlight`, or `DocumentPreview.PreviewDocument`.
2. Set `PdfjsLib.GlobalWorkerOptions.workerSrc` globally to the file path of the pdf.js worker script (see `src/setupTests.ts` for an example).
```

Always put `DocumentPreview` first, since that's our main component.

We should set up the example app to follow option (1), and then mention that here (like we do pointing to the tests for option (2)).